tests/unit/swagger/schemaDetails.spec.ts

```diff
@@ -127,4 +127,31 @@ describe('Schema details generation', () => {
       expect(errToTest!.message).to.eq(`Enums can only have string or number values, but enum MixedStringAndNumberEnum had number,string`);
     });
   });
+
+  describe('Extensions schema generation', () => {
```

Comment:
Can you please duplicate these tests to make sure this works for OpenAPI 3 aswell?

Reply:
@WoH Done.